code/web/src/modules/user/api/state.js

```diff
@@ -84,6 +84,11 @@ export const surveyReducer = (state = surveyInitialState, action) => {
         ...state,
         determinedStyle: action.payload
       } 
+    case 'SAVED_SURVEY_RESULTS':
+      return {
+        ...state,
```

Comment:
I'm happy to know how state works, seems to be a very powerful tool for front end